src/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.cs

```diff
@@ -5,6 +5,8 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
+[assembly: System.Runtime.CompilerServices.TypeForwardedTo(destination: typeof(System.Lazy<,>))]
```

Comment:
You are going to need to #ifdef this and include it only for netcoreapp, and for the other configurations you will need to include the API for `Lazy<T,M>` in the ref.

Reply:
Thanks @weshaggard

Reply:
Sorry your comment about netfx through me off but after looking at the configurations more closely you don't need to #ifdef this you should always have the type-forward here. 

As for the netfx build issue you need to fix it by adding a "_netfx" configuration in the src project for this library. However when you do that you will start having test failures because of https://github.com/dotnet/corefx/issues/25781 so you can either tackle that issue now or temporarily add a "_netfx" placeholder configuration in the test project as well.